src/firejail/util.c

```diff
@@ -45,6 +45,28 @@
 #define MAXBUF 4098
 #define EMPTY_STRING ("")
 
+int is_valid_unit(char c){
+	/* check char valid unit ( K or M or G ) */
+	if( c == 'K' || c == 'k' ) return 1;
+	if( c == 'M' || c == 'm' ) return 1;
+	if( c == 'G' || c == 'g' ) return 1;
+
+	return 0;
+}
+
+int unit_to_byte(char unit, long size){
```

Comment:
hm, you enter a `long`, but only return an `int`. they should have at least the same size.

Reply:
true. my bad.

Reply:
wait. i forgot to edit firejail.h . i fixed now.